app/scripts/lib/metaRPCClientFactory.test.js

```diff
@@ -86,13 +81,12 @@ describe('metaRPCClientFactory', function () {
     });
   });
 
-  it('should be able to handle notifications', function (done) {
+  it('should be able to handle notifications', () => {
```

Comment:
all tests where done was removed and no other mechanism introduced will always pass.